### DISCUSSION TITLE
Separate posts and boosts in home

### DISCUSSION THREAD

**Discussion by turbomoxxi** [mastodon/mastodon#26491] [General]
A quick search in the discussions and issues only brought up some issues from 6 years ago so I was wondering what the consensus was regarding posts/toots and boosts being clustered together. When I think of my own posts, I often wonder whether they'll end up drowned in a sea of boosts if I boost "too much". I don't clearly think in those terms but that's basically what it means. I've seen other people wonder the exact same thing online so I can't be the only one. Since Mastodon/Fediverse is all about boosting, I'm not sure whether bundling posts and boosts creates a situation where people hold back on boosting if they worry that their own content won't show up on their personal home page unless they pin some of them.

My thought was that perhaps separating them into "Posts", "Boosts", "Posts and replies", and "Media" would make more sense. I don't think I've seen that anywhere but please enlighten me if that already exists. I also like to be able to determine whether I should follow someone who, say, posts content I like but boosts things I don't care much about. If they boost a ton, I'll have a look at their profile and glance at 5 or 10 of their posts, which will happen to be boosts, conclude that I'm not interested. I'll miss out on their original content, which would have been interesting to me. In an ideal world, I would have followed that person but would have muted their boosts. If they reply a lot on content I don't care about, I guess the issue remains the same but I'd like the option to be able to see what original content they post without having to dig through boosts and replies.

**Comment by joric3** [comment]
Seconded. On a profile, I would like to be able to switch view between the complete timeline (posts & boosts) and reduced view (just posts, no boosts). 

Is there already a feature request for this?

Personally I make no use of the current feature to distinguish between "posts" and "posts & replies", but I understand that this feature request becomes more complicated if including/excluding replies among the options is kept. Similar for the "media" timeline: It is nice, but I don't really need it.
